core/geo/geoUtils.js

```diff
@@ -51,23 +56,68 @@ export const findSrsByCodeOrName = (codeOrName, limit = 200) =>
 
 const getSrsByCode = (code) => srsByCode[code]
 
+const toLatLon = (point) => {
+  if (!Point.isFilled(point)) return null
+
+  const srsId = Point.getSrs(point)
+  if (Srs.isLatLon(srsId)) {
+    // projection is not needed
+    return point
+  }
+  const x = Point.getX(point)
+  const y = Point.getY(point)
+
+  const srsFrom = getSrsByCode(srsId)
+  const srsTo = Srs.latLonSrs
+
+  const lonLat = proj4(
```

Comment:
latLng

Reply:
or better, [lon lat]; proj4 returns an array with x (longitude)  and y (latitude) coordinates)